src/Browser/WebStorage.purs

```diff
@@ -12,30 +14,35 @@ module Browser.WebStorage
   , setItem
   ) where
 
+  import Control.Monad.Eff
   import Data.Maybe
+  import Data.Function
+
+  foreign import data WebStorage :: !
+  type EffWebStorage eff = Eff (webStorage :: WebStorage | eff)
 
   class Storage s where
-    clear :: s -> s
-    getItem :: forall v. s -> String -> Maybe v
-    key :: s -> Number -> Maybe String
-    length :: s -> Number
-    removeItem :: s -> String -> s
-    setItem :: forall v. s -> String -> v -> s
+    clear :: forall eff. s -> EffWebStorage eff Unit
+    getItem :: forall eff. s -> String -> EffWebStorage eff (Maybe String)
+    key :: forall eff. s -> Number -> EffWebStorage eff (Maybe String)
+    length :: forall eff. s -> EffWebStorage eff Number
+    removeItem :: forall eff. s -> String -> EffWebStorage eff Unit
+    setItem :: forall eff. s -> String -> String -> EffWebStorage eff Unit
```

Comment:
Why the change to return `Unit`? Is it because the api doesn't return anything to begin with? I'm fine with the change. Just wanted to understand it.